### PR TITLE
Thiruvenkatam A1,A2 and B4

### PR DESCRIPTION
**Pull request by ThiruvenkatamR**
I have pushed the UI assignments. Can you please merge this?

### CODE REVIEW

File: thiru/boxmodel.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    margin-bottom: 25px;
+}
+
+.overflow-scroll {
+    height: 60px;
+    overflow-y: scroll;
+    border: 2px black solid;
+    margin-bottom: 25px;
+}
+.footer{
```
Comment (iammadhan) [inline thiru/boxmodel.css]: why a new class for footer?

File: thiru/boxmodel.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.wrapper{
+    width:100%;
+    height:97vh;
+    margin:0 auto;
+    box-sizing: border-box;
+    padding: 50px;
+    border: 5px solid black;
+}
+.header {
```
Comment (iammadhan) [inline thiru/boxmodel.css]: What is the need for a new class header?

File: thiru/venn.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<!-- <!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js">
```
Comment (iammadhan) [inline thiru/venn.html]: Why this no-js class?

File: thiru/design3.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <meta charset="utf-8">
+    <meta name="keywords" content="design">
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1"> -->
+    <link rel="stylesheet" type="text/css" href="design.css">
+</head>
+
+<body>
+    <header>Header</header>
+    <div class="sidebar-1">Sidebar 1</div>
+    <article class="article">Article</article>
```
Comment (iammadhan) [inline thiru/design3.html]: Reason for "article" class?

Comment (ThiruvenkatamR) [reply thiru/design3.html]: Sorry will remove it

File: thiru/venn.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+.circle.circle3 {
+    left: 85px;
+    top: 150px;
+    background-color: #f5b7b1;
+    border: solid red;
+}
+
+.circle {
+    pointer-events: auto;
```
Comment (iammadhan) [inline thiru/venn.css]: why pointer-events: auto

File: thiru/venn.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+<head>
+    <meta charset="utf-8">
+    <title>venn</title>
+    <meta name="description" content="venn">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=0, maximum-scale=1.0">
+    <link rel="stylesheet" href="venn.css">
+</head>
+
+<body>
+    <div class="wrapper">
```
Comment (iammadhan) [inline thiru/venn.html]: why is this wrapper div?

File: thiru/venn.css (hunk 5 above)
Comment (ThiruvenkatamR) [reply thiru/venn.css]: removed

File: thiru/venn.html (hunk 6 above)
Comment (ThiruvenkatamR) [reply thiru/venn.html]: removed

**Pull request merged**